Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM public.ecr.aws/docker/library/python:3.9-slim-bookworm
+
+# Prevent Python from writing .pyc files
+ENV PYTHONDONTWRITEBYTECODE=1
+
+# Install system dependencies (curl, gnupg, Node.js)
+RUN apt-get update && apt-get install -y curl gnupg && \
+    curl -fsSL https://deb.nodesource.com/setup_22.x | bash - && \
+    apt-get install -y nodejs && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# Set working directory
+WORKDIR /app
+
+# Install Python project dependencies (globally in the image)
+RUN pip install pyyaml
+
+# Copy package.json and install Node.js project dependencies into /app/node_modules
+COPY package.json ./
```

Comment:
I don't know if you intend for these to work out the box, but the repo doesn't have a `package.json` by default so I had to `npm init` then `npm install` to get the `package-lock.json` as well.

The comments on lines #21 and #22 I had to remove as well since I don't think those are treated as comments in that way.